client/lib/domains/assembler.js

```diff
@@ -31,6 +31,7 @@ export function createDomainObjects( dataTransferObject ) {
 			expirySoon: domain.expiry_soon,
 			googleAppsSubscription: assembleGoogleAppsSubscription( domain.google_apps_subscription ),
 			hasPrivacyProtection: domain.has_private_registration,
+			hasWpcomNameservers: domain.has_wpcom_nameservers,
```

Comment:
What about the `transfer_status` flag from the backend patch?

Also, when adding new fields to this assembler, please remember to update the Redux counterpart: https://github.com/Automattic/wp-calypso/blob/4d635956810bed5dd0cc484a9485c698e546e401/client/state/sites/domains/assembler.js#L10

Reply:
Ah....ok. Good. to. know. Thanks.